### DISCUSSION TITLE
How to find a map

### DISCUSSION THREAD

**Discussion by Herve-be** [mapsforge/mapsforge#1459] [Q&A] [answered]
Hi,
I can't figure out how to find a map.
For example, I'm looking for a map of the Cayman Islands : which file is it in?
Same question for Puerto Rico.
Thank you in advance for your help

**Comment by devemux86** [comment]
Mapsforge maps are created with the [Geofabrik](https://download.geofabrik.de/) structure.

Maps can be found by continent / country if their data also exist in Geofabrik.

**Comment by devemux86** [comment]
There are also other map providers to look for offline Mapsforge maps:

- [Mapsforge map providers](https://github.com/mapsforge/mapsforge/blob/master/docs/Mapsforge-Maps.md)

**Comment by Herve-be** [comment]
Thanks for your replies but I still can't find the maps for Cayman Islands and Puerto Rico.

**Reply by devemux86** [reply, marked as answer]
Check [OpenAndroMaps](https://www.openandromaps.org/en/), in South and Middle America section they have several maps, for example Caribbean Sea.

**Comment by Herve-be** [comment]
On OpenAndroMaps Caribbean Sea Coverage I see Cayman Islands and Puerto Rico : OK
https://www.openandromaps.org/wp-content/images/mapsV5/southamerica/CaribbeanSea.jpg

I've downloaded the map but I don't find it among OruxMaps off-line maps.
When I download a MapsForge Map I see it immediately after download.
Which action is missing ?

**Reply by devemux86** [reply]
> I've downloaded the map but I don't find it among OruxMaps off-line maps.

For this better ask in Orux or OpenAndroMaps forum.

**Comment by Herve-be** [comment]
Done on OpenAndroMaps forum, thanks.
